app/javascript/components/Identifier/Identifier.tsx

```diff
@@ -42,7 +43,7 @@ const StyledUswdsIdentifier = styled(UswdsIdentifier).attrs<{ styles: FontsAndCo
 `;
 
 // eslint-disable-next-line complexity
-export const Identifier = ({ identifierContent, identifierLinks }: IdentifierProps) => {
+export const Identifier = ({ identifierContent, identifierLinks, showVoteOrgLink }: IdentifierProps) => {
```

Comment:
Can we add a `false` to `showVoteOrgLink` as a default so we don't have to pass it all the time.

Reply:
PR is updated.